### DISCUSSION TITLE
part-ii/2010-62/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#291] [Question comments]
Part II 2010 Galois Theory: Paper 3, Section II, H 

https://questions.tripos.org/part-ii/2010-62/

**Comment by DannyDuberstein** [comment]
Don't know why structure theorem is relevant in (ii)?? Also why "succession" in the first sentence? Don't you just say Aut(F(mu)/F) injects into (Z/nZ)* so is cyclic of order at most n-1, F has prim root of that order so extension is Kummer.
